fem/bilininteg_mass.cpp

```diff
@@ -300,43 +300,47 @@ static void SmemPAMassApply2D(const int NE,
 {
    const int D1D = T_D1D ? T_D1D : d1d;
    const int Q1D = T_Q1D ? T_Q1D : q1d;
-   const int MQ1 = T_Q1D ? T_Q1D : MAX_Q1D;
-   const int MD1 = T_D1D ? T_D1D : MAX_D1D;
+   constexpr int NBZ = T_NBZ ? T_NBZ : 1;
+   constexpr int MQ1 = T_Q1D ? T_Q1D : MAX_Q1D;
```

Comment:
Why do we need both `const int` and `constexpr int`?

Reply:
Using `constexpr` makes it clear that these are compile-time constants. With `const` it is not clear immidiately. Some of the other `const int` in these kernels can be replaced with `constexpr` but not all.